common/invoke.go

```diff
@@ -0,0 +1,31 @@
+package common
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/interop/crypto"
+	"github.com/nspcc-dev/neo-go/pkg/interop/runtime"
+)
+
+func InvokeID(args []interface{}, prefix []byte) []byte {
+	for i := range args {
+		arg := args[i].([]byte)
+		prefix = append(prefix, arg...)
+	}
+
+	return crypto.SHA256(prefix)
+}
+
+type IRNode struct {
```

Comment:
Let's move inner ring list related code to `ir.go`: both `IRNode` struct and `InnerRingInvoker` function.